src/main/java/org/fcrepo/spec/testsuite/App.java

```diff
@@ -98,6 +98,7 @@ public static void main(final String[] args) {
 
         //Create the default container
         final Map<String, String> params = new HashMap<>();
+        params.put("param0", inputUrl);
```

Comment:
I am wondering if we can assume that inputUrl is the root of the repository (that seems to be the intended use of this parameter). A user might specify a subcontainer to run the tests in rather than the root? In fcrepo-import-export we go through the trouble of seeking the root explicitly https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/main/java/org/fcrepo/importexport/exporter/Exporter.java#L442-L449

Reply:
@bbpennel : I'm wondering if we should make the command line docs explicit - ie that the baseURL is the root, remove param0,  and change param1 to use the rootURI rather than create a separate test container since most if not all the tests create child test resources anyway.  What do you think?

Reply:
I think the other purpose of the test container is to keep the test resources together in one place rather than cluttering up the root, so I'd rather not get rid of it. Declaring in the docs that baseURL has to be the root is probably fine, if that restriction is okay.

I feel a little weird about singling out the root solely for the purpose of getting an uncontained resource, but i'm not sure what else we could use. LDPCvs are uncontained (sort of), but they operate differently as far as ACLs go.

Reply:
Also, the lack of containment is implementation specific.  I believe it would be reasonable for others to be explicit about contains between the original and the ldpcv.